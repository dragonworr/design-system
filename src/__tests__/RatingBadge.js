import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import { RatingBadge } from '..'

describe('RatingBadge', () => {
  test('renders', () => {
    const json = renderer.create(<RatingBadge />).toJSON()
    expect(json).toMatchSnapshot()
  })
})
