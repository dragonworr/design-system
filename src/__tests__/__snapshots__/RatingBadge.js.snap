// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RatingBadge renders 1`] = `
.c1 {
  padding-left: 16px;
  padding-right: 16px;
  color: white;
  background-color: orange;
}

.c0 {
  display: inline-block;
  line-height: 1.5;
  font-weight: bold;
  border-radius: 2px;
}

<div
  className="c0 c1"
  color="white"
  fontWeight="bold"
/>
`;
